assets/app/styles/_forms.less

```diff
@@ -0,0 +1,15 @@
+//
+// Forms
+// ------------------------------------------------------
+
+// Prevent zooming when focusing input fields in mobile Safari. Try to only
+// target mobile devices to avoid larger-than-wanted fonts elsewhere.
+@media only screen and (max-device-width: 736px) and (-webkit-min-device-pixel-ratio: 0) {
+  input[type="text"],
+  input[type="number"],
+  input[type="search"],
+  input[type="url"],
+  textarea {
+    font-size: 16px;
+  }
+}
```

Comment:
Looks like you've lost some changes on rebase.

```
// Prevent zooming when focusing input fields in mobile Safari. Try to only
// target mobile devices to avoid larger-than-wanted fonts elsewhere.
@media only screen and (max-device-width: 736px) and (-webkit-min-device-pixel-ratio: 0) {
  input[type="text"],
  input[type="number"],
  input[type="search"],
  input[type="url"],
  textarea,
  .modal-resource-edit .editor {
    font-size: 16px;
  }

  // Fix Bootstrap modal dialog zoom on the iPhone
  // See http://stackoverflow.com/questions/32675849/screen-zooms-in-when-a-bootstrap-modal-is-opened-on-ios-9-safari
  body {
    padding-right: 0px !important
  }
  .modal-open {
    overflow-y: auto;
  }
}
```